### PR TITLE
9 presenter game presenter with output boundary

### LINKED ISSUE #9: Presenter: Game Presenter and its output boundary
Presenter for the game that is used by the view model

Responsibility:
- Translates data from the output boundary to be usable by ViewModel

- Gives only the information required to produce a view to the ViewModel

### PR DESCRIPTION
**Pull request by NotSakura**
Presenter for game and output boundary for it

### CODE REVIEW

**Review by ivyparc — CHANGES_REQUESTED** [review]
will need a change in the structure inside UpdateGame method

File: src/main/java/GamePresenter.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        // another way is for the presenter to return something but that would mean the main menu takes care of the
+        // timing which is a violation of CA
+        ArrayList<String> info = new ArrayList<String>();
+        info.add(position);
+        info.add(Integer.toString(time));
+        info.add(Integer.toString(score));
+        this.viewModel.updateGame(info);
+
+        // This will call on view model to set the data (trusting that view will get that data from view model)
+        this.viewModel.updateGame(position, time, score);
```
Comment (ivyparc) [inline src/main/java/GamePresenter.java]: The assignment should be info

**Review by ivyparc — DISMISSED** [review]
Nice change in updateGame method. Processing data here before passing to viewModel is a good choice.

**Review by 123HenryLiu — APPROVED** [review]
Fixed the view model attribute

**Review by Cathynvrdie — APPROVED** [review]
cool.

**Review by ivyparc — APPROVED** [review]
Fix resolved

**Pull request merged**